src/client/app/App.tsx

```
@@ -277,117 +312,307 @@ const LandingHero = ({
                             variant="ghost"
-                            className="w-full h-14 text-lg font-bold border-2 border-[#D0B378]/70 text-[#EBD4A0] hover:bg-[#D0B378] hover:text-[#1D1F2B] bg-transparent transition-all"
+                            className="w-full h-12 font-bold uppercase tracking-widest rounded-none"
+                            style={{
+                                border: '1px solid rgba(0,229,255,0.35)',
+                                color: 'rgba(0,229,255,0.75)',
+                                fontFamily: 'monospace',
+                                letterSpacing: '0.15em',
+                            }}
                         >
-                            JOIN EXISTING GAME
+                            // JOIN EXISTING GRID
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)